charts/schema-registry/templates/configmap.yaml

```diff
@@ -28,13 +28,19 @@ data:
   "pravega_client_auth_method": "{{ .Values.client.method }}"
   "pravega_client_auth_token": "{{ .Values.client.token }}"
   {{- if .Values.authentication.enabled }}
+  {{- if not .Values.authentication.disableBasicAuthentication }}
```

Comment:
should we also check if auth is enabled? otherwise the password file is not required.

Reply:
The control reaches this line only if auth has been enabled, which is verified in the previous line